menyer/menyer/emmies-restaurang/index.html

```diff
@@ -0,0 +1,88 @@
+            <div class="food-item"></div>
+            <img src="https://assets.icanet.se/q_auto,f_auto/imagevaultfiles/id_161301/cf_259/toast-med-currykyckling-och-picklad-silverlok-7221.jpg"
+                width="400" alt="Toast med kycklingcurryröra">
+            </p>
+            <p>
+                <strong>Kycklingtoast</strong> - <i>Frasig fullkorns toast med vår berömda kycklingcurry röra</i> 
+            </p>
+            <p class="food-item-price"> <strong>15 kr</strong> </p>
+            </div>
+        </section>
+        <section id="main-courses">
+            <h2>Huvudrätter</h2>
+            <div class="food-item">
+                <img src="https://thumbs.dreamstime.com/b/schnitzel-fr%C3%A5n-%C3%B6ver-87429749.jpg" width="400"
+                    alt="Schnitzel med pommes och bearnaisesås">
+                <p>
+                    <strong>Schnitzel med pommes</strong> - <i>Krispig scnitzel med ännu krispigare pommes, med krämig bearnaisesås bredvid</i> 
+                </p>
+                <p class="food-item-price"><strong>124 kr</strong></p>
+            </div>
+            <div class="food-item">
+                <img src="https://cdn.loveandlemons.com/wp-content/uploads/2020/02/alfredo-sauce.jpg" width="400"
+                    alt="Fettuccini alfredo">
+            </div>
+            <div>
+                <p><strong>Fettuccini alfredo</strong> - <i>Med egengjord fettuccini</i>
+                </p>
+                <p class="food-item-price"><strong>130 kr</strong>
+                </p>
+            </div>
+        </section>
+        <section id="deserts">
+            <h2>Efterrätter</h2>
+            <div class="food-item">
+                <img src="https://www.simplyrecipes.com/wp-content/uploads/2018/02/Creme-Brulee-LEAD-VERTICAL.jpg" width="400" alt="crème brûlée">
+                <p>
+                    <strong>Vår knäckiga crème brûlée</strong> - <i>En smakupplevelse du sent kommer glömma</i> 
+                </p>
+                <p class="food-item-price"><strong>79 kr</strong></p>
+            </div>
+            <div class="food-item">
+                <img src="https://img.koket.se/standard-large/saftig-karleksmums-i-langpanna.jpg" width="400"
+                alt="Kärleksmums">
+
+                <p>
+                    <strong>Kärleksmums</strong> - <i>Farmors recept, kan du gissa den hemliga ingrediensen får du den på
+                    köpet</i> 
+                </p>
+                <p class="food-item-price"><strong>59 kr</strong></p>
+            </div>
+        </section>
+    </main>
+    <footer>
+        <p class="menu-info">
+            <a class="return-link" href="/restaurang-menyer/menyer/exempel/index.html">Hem</a>
+        </p>
+        <p class="menu-info">
+            <strong>Alla våra alternativ går att få vegetariska</strong>
+            <br>
+            <small>Författad av Emmie Sjunnesson, September, 2020</small>
+        </p>
+    </footer>
+
+</body>
+
+</html>
```

Comment:
Try to always end you find with a new line (Press enter one more time). Some programs that analyze your code will not work as well otherwise.